update(multiple): onlySpinnakerManaged Additional information

kinnonyee-armory opened the pull request:
updated to address possible issues, and to clarify the "automated" nature of setting flag to false

Please note, Document should be reviewed by @dwilliams-armory before publishing

kinnonyee-armory committed:
update(multiple): onlySpinnakerManaged Additional information

updated to address possible issues, and to clarify the "automated" nature of setting flag to false

317brian commented on content/en/docs/armory-admin/kubernetes-account-add.md:
```diff
@@ -21,7 +21,7 @@ At a high level, Spinnaker operates in the following way when deploying to Kuber
   * `name`: A Spinnaker-internal name used to refer to the Kubernetes account.  This is the item you will select in various Kubernetes stages to indicate that you would like to deploy to this particular Kubernetes account (which again, consists of a Kubernetes cluster/credential/context set)
   * `kubeconfigFile`: A file path referencing the contents of the kubeconfig file for connecting to the target cluster. When using the Operator, this can be any name that should match an entry in `spec.spinnakerConfig.files` where the file contents is copied. If using Halyard, this is a physical file path. This field supports referencing files stored in external [secret engines]({{< ref "secrets" ...
-  * `onlySpinnakerManaged`: A boolean that indicates whether Spinnaker should only manage / display entities that were created by Spinnaker (i.e., those that have the Spinnaker labels and annotations that indicate that they were created by Spinnaker)
+  * `onlySpinnakerManaged`: When true, Spinnaker will only cache/display applications that have been created by Spinnaker.  Before placing in a "false" state, it is recommended to review the Kubernetes cluster configuration.  Once applied with a false flag, Spinnaker will analyze the cluster and automatically attempt to configure and populate applications for resources already present in Kubernet ...
```
@dwilliams-armory this is the relevant part. Please ignore all the warnings/errors. We just installed a Linter and are working on tuning it. 🙇🏼‍♂️

aimeeu committed:
Merge branch 'master' into kinnon-onlyspinnakermanaged

aimeeu committed:
copy edits

aimeeu commented on content/en/docs/armory-admin/kubernetes-account-add.md:
```diff
@@ -21,7 +21,7 @@ At a high level, Spinnaker operates in the following way when deploying to Kuber
   * `name`: A Spinnaker-internal name used to refer to the Kubernetes account.  This is the item you will select in various Kubernetes stages to indicate that you would like to deploy to this particular Kubernetes account (which again, consists of a Kubernetes cluster/credential/context set)
   * `kubeconfigFile`: A file path referencing the contents of the kubeconfig file for connecting to the target cluster. When using the Operator, this can be any name that should match an entry in `spec.spinnakerConfig.files` where the file contents is copied. If using Halyard, this is a physical file path. This field supports referencing files stored in external [secret engines]({{< ref "secrets" ...
-  * `onlySpinnakerManaged`: A boolean that indicates whether Spinnaker should only manage / display entities that were created by Spinnaker (i.e., those that have the Spinnaker labels and annotations that indicate that they were created by Spinnaker)
+  * `onlySpinnakerManaged`: When `true`, Spinnaker only caches and displays applications that have been created by Spinnaker.  Before placing in a `false` state, you should review the Kubernetes cluster configuration.  When `false`, Spinnaker analyzes the cluster and automatically attempts to configure and populate applications for resources already present in Kubernetes, unless limited with `omi ...
```
waiting for Daniel Williams to review

dwilliams-armory replied:
This looks good, however the zombie applications aren't unique to MySQL.  They happen with Redis too.  MySQL just has a second issue where the cache doesn't clean up, so the zombie apps don't go away when the setting is set back to true.  Also, might be a good idea to use "auto-generated application placeholders" or similar instead of "zombie applications".

kinnonyee-armory committed:
update(Kubernetes-account): Add Daniel Suggestions

Made changes to the information so it reflects Daniel's suggestions for wording and accuracy

kinnonyee-armory committed:
update(providers): Incorporate Daniel's Suggestions

Changed description of possible issue to more accurately reflect Daniel's suggestions

kinnonyee-armory commented:
@aimeeu I've adjusted the wording to match Daniel's request.

aimeeu committed:
Merge branch 'master' into kinnon-onlyspinnakermanaged
